### PR TITLE
Fix grammar in Custom Error Pages guide

### PR DESCRIPTION
**Pull request by roryokane**
And improve wording slightly.

### CODE REVIEW

**Comment by jodosha** [comment]
@cllns Would you mind to check this? Thanks!

File: source/guides/views/custom-error-pages.md [hunk 1]
```diff
@@ -4,20 +4,18 @@ title: Guides - Custom Error Pages
...
+When an unsuccessful request is returned, there are some special pages that a Hanami application presents to users.
+These pages have a generic graphic and some basic information like the HTTP status code and the message.
 
-Hanami allows to customize them on an application basis.
-We need to create a template with the corresponding HTTP code as filename (eg. `apps/web/templates/500.html.erb`).
-
-Since now on, all the side errors will be presented using that template.
+Hanami allows us to customize them on a per-application basis.
+We just need to create a template with the corresponding HTTP code as the filename (e.g. `apps/web/templates/500.html.erb`).
+From then on, all the side errors will be presented using that template.
```
Comment (cllns) [inline source/guides/views/custom-error-pages.md]: Can you fix 'all the side errors' too? I'm not completely sure what that means. Instead maybe say something about how 500 is a system error (like an uncaught exception)?

**Comment by jodosha** [comment]
@cllns Feel free to merge and possibly fix yourself with minor cleanups. 😉

**Comment by cllns** [comment]
👍 will do

File: source/guides/views/custom-error-pages.md (hunk 1 above)
Comment (roryokane) [inline source/guides/views/custom-error-pages.md]: For the record, I think “side errors” was supposed to be “server-side errors”. But you already improved that section’s wording to be more even more specific, in be0e557b8f6e4c35dc7435fb741dbf03239b5340.

Comment (cllns) [inline source/guides/views/custom-error-pages.md]: Ah, that makes sense! Thanks again for this fix :)

**Comment by jodosha** [comment]
@roryokane @cllns Thank you folks!

**Pull request merged**